.codegen/service.go.tmpl

```diff
@@ -113,7 +113,7 @@ func New() *cobra.Command {
 // Functions can be added from the `init()` function in manually curated files in this directory.
 var {{.CamelName}}Overrides []func(
 	*cobra.Command,
-	{{- if .Request }}
+	{{- if not .IsLegacyEmptyRequest }}
```

Comment:
Can you explain the semantics of `.IsLegacyEmptyRequest` in the PR description? For which fields will it be set.